paywall/src/paywall-script/index.ts

```diff
@@ -1,14 +1,31 @@
 import Postmate from 'postmate'
+import {
+  CheckoutEvents,
+  UserInfo,
+  TransactionInfo,
+} from '../../../unlock-app/src/hooks/useCheckoutCommunication'
```

Comment:
This turns out not to be kosher in the docker images, since this file is never copied over. I'd like to avoid duplicating this code, but I'll have to figure out the best way to do it.